spdx-utils/src/main/kotlin/SpdxLicenseAliasMapping.kt

```diff
@@ -23,7 +23,8 @@ import com.here.ort.spdx.SpdxLicense.*
 import com.here.ort.spdx.SpdxLicenseException.*
 
 /**
- * A mapping from varied SPDX license names to valid SPDX expressions.
+ * A mapping from varied SPDX license names to valid SPDX expressions. When mapping a name without any indication of a
+ * version to an SPDX expression with a version, the most commonly used version at the time of writing is used.
```

Comment:
How is it determined what the "most commonly used version" is? Didn't we decide last week to remove such ambiguous mapping?

Reply:
> How is it determined what the "most commonly used version" is?

Good question, it's probably hardly possible to come up with an objective metric here. I just tried to come up with a rationale for @tsteenbe's changes, so at least document the intent somehow.

> Didn't we decide last week to remove such ambiguous mapping?

Hmm, I can't remember to have even discussed this. Removing all mappings from a name without a version to SPDX IDs with a version would affect quite some (already existing) entries, so I'm not sure how much it would break backwards-compatibility.

Reply:
Let's discuss this offline and merge this as is, as it is consistent with how we mapped licenses so far.